strings.txt

```diff
@@ -12094,17 +12094,17 @@ THOUSANDS_SEP
 	DE	.
 	EN	,
 	ES	.
-	FI	 
```

Comment:
Hmm... Your editor seems to automatically remove all trailing whitespace. While I don't like it in general as it creates "empty" changes, in this case it's even wrong: the value for FI and PL should be a tab followed by a space. Could you please revert these?